src/rgw/rgw_common.cc

```diff
@@ -201,7 +201,7 @@ static string get_abs_path(const string& request_uri) {
   return request_uri.substr(beg_pos, len - beg_pos);
 }
 
-req_info::req_info(CephContext *cct, class RGWEnv *e) : env(e) {
+req_info::req_info(CephContext *cct, class RGWEnv *_env) : env(_env) {
```

Comment:
Just a suggestion: maybe `req_info::env` could be turned into a pointer to constant. If I see correctly the methods of `req_info` call only `RGWEnv::get`.